BookPlayer/Library/RootViewController.swift

```diff
@@ -91,28 +105,102 @@ class RootViewController: UIViewController {
         })
     }
 
-    @objc func bookChange(_ notification: Notification) {
+    @objc private func bookChange(_ notification: Notification) {
         guard let userInfo = notification.userInfo,
             let books = userInfo["books"] as? [Book],
             let currentBook = books.first else {
                 return
         }
 
-        setupFooter(book: currentBook)
+        setupMiniPlayer(book: currentBook)
 
         PlayerManager.shared.play()
     }
 
-    @objc func bookReady(_ notification: Notification) {
+    @objc private func bookReady(_ notification: Notification) {
         guard let userInfo = notification.userInfo,
             let book = userInfo["book"] as? Book else {
                 return
         }
 
-        setupFooter(book: book)
+        setupMiniPlayer(book: book)
+    }
+
+    @objc private func bookDeleted(_ notification: Notification) {
```

Comment:
it seems like the notification `bookDeleted` isn't being actually used

Reply:
😯 oh right, I forgot the second part of the ticket. Deleting a book should stop the book as well.